source/docs/guides/3D_secure_2_0.md

```diff
@@ -612,6 +614,9 @@ All the browser values can be collected via JavaScript, unless of course they ha
 |card.number|String|The credit card number to authenticate against|
 |card.expiry.month|Number|The two digit month of the card expiration date.|
 |card.expiry.year|Number|The four digit year of the card expiration date.|
+|card.name|String|Name of the cardholder *Mandatory for Visa|
```

Comment:
should we say: Name of the cardholder is *mandatory for Visa

Reply:
No I don't think so, "Name of the cardholder" and "Mandatory for Visa" are 2 different statements, the second is an additional note on the end of the description